# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/svgalib/installmodule.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.cfg.orig	2003-01-24 13:56:05.000000000 +0200
+++ ./Makefile.cfg	2003-10-06 21:52:14.000000000 +0300
@@ -128,6 +128,9 @@
 # Comment this out if you use devfs only and don't want to make svga nodes
 INSTALLDEV = installdev
 
+# Comment this out if you are going to install the kernel module manually
+INSTALLMODULE = installmodule
+
 #
 # Comment out any driver that you don't want included in the library.
 #
--- ./Makefile.orig	2002-12-22 12:58:47.000000000 +0200
+++ ./Makefile	2003-10-06 21:52:15.000000000 +0300
@@ -211,7 +211,7 @@
 	(cd threeDKit/; $(MAKE) install)
 	 
 install: uninstall installheaders $(INSTALLSHAREDLIB) installconfig \
-	$(INSTALLSTATICLIB) $(INSTALLUTILS) $(INSTALLMAN) installmodule $(INSTALLDEV) \
+	$(INSTALLSTATICLIB) $(INSTALLUTILS) $(INSTALLMAN) $(INSTALLMODULE) $(INSTALLDEV) \
 	lib3dkit-install
 	@echo
 	@echo
